ui/vercel.json

```diff
@@ -0,0 +1,12 @@
+{
+  "github": {
+    "silent": true
```

Comment:
This turns off comments like [this](https://github.com/hashicorp/nomad/pull/9471#issuecomment-735883596).